doc/website/getting-started/installation.md

```diff
@@ -64,103 +64,113 @@ QNX SDP 7.0 and 7.1 are supported (shipping with gcc 5.4 and gcc 8.3 respectivel
-
+    
+    ```bash
+    cmake --build build
+    ```
     !!! tip
         You can fasten up the build by appending `-j 4` where 4 stands for the number of parallel build processes.
         You can choose more or less depending on your available CPU cores on your machine.
 
 4. Install to system
 
     Mac:
-
-        cmake --build build --target install
-
+    ```bash
+    cmake --build build --target install
+    ```
     Linux:
-
-        sudo cmake --build build --target install
-
+    ```bash
+    sudo cmake --build build --target install
+    ```
     !!! tip
         The installation directory is usually left at its default, which is `/usr/local`
 
     !!! note
-        iceoryx is build in Release mode with `-O3` optimization by default. If you want to have     debug symbols please
+        iceoryx is build in Release mode with `-O3` optimization by default. If you want to enable debug symbols please
         set `CMAKE_BUILD_TYPE=Debug`.
 
 ### Build options
 
-Please take a look at the CMake file [build_options.cmake](../../../iceoryx_meta/build_options.cmake) to get an overview of the available build options for enabling additional features.
-
-### Available CMake switches you can customize for the iceoryx_posh build
-
- |  switch  |  description |
- |:---------|:-------------|
- | `IOX_MAX_PUBLISHERS` | the maximum number of publishers one `RouDi` instance can manage |
- | `IOX_MAX_SUBSCRIBERS_PER_PUBLISHER` | the maximum number of subscriber a publisher can deliver chunks to|
- | `IOX_MAX_PUBLISHER_HISTORY` | the maximum number chunks available for the publisher history |
- | `IOX_MAX_CHUNKS_ALLOCATED_PER_PUBLISHER_SIMULTANEOUSLY` | the maximum number of chunks a publisher can allocate at a given time |
- | `IOX_MAX_SUBSCRIBERS` | the maximum number of subscribers one `RouDi` instance can manage |
- | `IOX_MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY` | the maximum number of chunks a subscriber can hold at a given time |
- | `IOX_MAX_INTERFACE_NUMBER` | the maximum number for interface ports, which are used for e.g. gateways |
-
-Have a look at [iceoryx_posh_deployment.cmake](../iceoryx_posh/cmake/iceoryx_posh_deployment.cmake) for the default values of this constants.
+Please take a look at the CMake file [build_options.cmake](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_meta/build_options.cmake) to get an overview of the available build options for enabling additional features.
 
 ## Build with script
 
-As an alternative we provide our build-test script which we use to integrate iceoryx into our infrastructure.
-The intention of the script is to more than just building with iceoryx. This is for doing a code coverage scan or for using the adress-sanitizer.
+As an alternative we provide a build-test script which we use to integrate iceoryx into our infrastructure.
+The intention of the script is more than just building with iceoryx. This is for doing a code coverage scan or using the adress-sanitizer.
```

Comment:
The intention of the script goes beyond building iceoryx, it is also used for the code coverage scan or the address-sanitizer runs on the CI.